modules/react/tabs/lib/TabsList.tsx

```diff
@@ -44,12 +44,12 @@ export const useTabsList = composeHooks(
 );
 
 const StyledStack = styled(Stack)<StyledType>({
+  maskImage: `linear-gradient(to right, white 95%, transparent)`,
```

Comment:
Not sure if this is meant for the next major, but [mask-image doesn't have IE11 support](https://caniuse.com/?search=mask-image).

Reply:
This is okay since this should only apply to touch modality. It is possible a touch device supports IE11, but we're okay with the mask image not working in this case.